test/box/mvcc_mass.test.lua

```diff
@@ -0,0 +1,423 @@
+test_run = require('test_run').new()
+netbox = require('net.box')
+console = require('console')
+yaml = require('yaml')
+json = require('json')
+--------------------------------------------------------------------------------
+
+-- False iff only successful txns included in reprocucer
+is_reproduce_all = false
+
+-- START HOST
+test_run:cmd('create server mvcc_mass with script="box/mvcc_mass.lua"')
+test_run:cmd('start server mvcc_mass')
+--------------------------------------------------------------------------------
+
+-- START HOST INIT
+test_run:cmd('switch mvcc_mass')
+box.schema.user.grant('guest','read,write,execute,create,drop','universe')
+
+txn_proxy = require('txn_proxy')
+
+-- simultaneous txns
+sim = box.schema.space.create('s', {engine='memtx', if_not_exists=true})
+sim_tree = sim:create_index('st', {type='tree'})
+sim_hash = sim:create_index('sh', {type='hash'})
+
+-- START HOST API DEFINITIONS
+test_run:cmd('setopt delimiter ";"')
+math.randomseed(os.time());
+
+-- Probability of the rollback
+rollback_prob = 0.05
+
+-- Probability of the commit
+commit_prob = 0.2
+
+-- Maximum random key value (min is 0)
+max_key_val = 100
```

Comment:
I would propose to set this to 10. I checked and saw that with 100 the majority of selects has empty results.

Reply:
Fixed.